test/Cli.FunctionalTests/Cli.FunctionalTests.csproj

```diff
@@ -1,12 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
I would prefer to keep this targeting `netcoreapp2.1`, so the same test project can be used to verify both `2.1.*` and `2.2.*` CLIs.